### PR TITLE
docs(readme): include interpolation example in usage

### PR DESCRIPTION
**Pull request by portikM**
# Summary

## PR Checklist

* [ ] **Tests pass:** check the output of all package unit and/or component tests.
  * [ ] If this PR is the result of a bug, test coverage was added accordingly.
* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/shared-ui-components#committing-changes).
* [ ] **Docs:** includes a technically accurate README, and the docs have been updated accordingly based on the changes in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,8 @@ As an example, in Kong's [Kongponents](https://kongponents.konghq.com) Vue compo
 @import "@kong/design-tokens/tokens/scss/variables";
 
 .my-component-class {
+  --my-css-variable: var(--kui-space-10, #{$kui-space-10});
+  margin: var(--my-css-variable);
```
Comment (adamdehaven) [inline README.md:167]: Can you provide a fallback value here

File: README.md [hunk 2]
```diff
@@ -163,6 +163,8 @@ As an example, in Kong's [Kongponents](https://kongponents.konghq.com) Vue compo
 @import "@kong/design-tokens/tokens/scss/variables";
 
 .my-component-class {
+  --my-css-variable: var(--kui-space-10, #{$kui-space-10});
```
Comment (adamdehaven) [inline README.md:166]: Add a note here
```suggestion
  // When setting a CSS variable value to a SCSS fallback, you need to interpolate the SCSS variable
  // Interpolation can be used almost anywhere in a Sass stylesheet to embed the result of a SassScript expression into a chunk of CSS. Just wrap an expression in #{}
  --my-css-variable: var(--kui-space-10, #{$kui-space-10});
```

File: README.md [hunk 3]
```diff
@@ -243,10 +247,10 @@ yarn install --frozen-lockfile
...
+    | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | `/tokens/alias`  | The `alias` directory **must** only contain alias values that point directly to a raw CSS value. Any tokens defined within the `alias` directory **will not** be exposed in the package exports. Tokens defined in the `/tokens/alias/` directory can be utilized/referenced within the `/tokens/source/` files; however, these tokens will **NOT** be exported in the build files. |
+    | `/tokens/source` | The `source` directory contains all tokens that **will be** available for consumption from the package exports.                                                                                                                                                                                                                                                                     |
```
Comment (adamdehaven) [inline README.md:253]: Can you back out this change please

File: README.md [hunk 4]
```diff
@@ -176,6 +178,8 @@ Inspecting the example above, you will notice that we fist import the SCSS varia
 
 When Kongponents are imported and used in a host application, the components will utilize the SCSS fallback values by default since the CSS variables are undefined. This is the normal usage and works great for most applications.
 
+**Note:** notice how you have to use [interpolation](https://sass-lang.com/documentation/interpolation/) when using SCSS tokens in custom property values (such as defining your CSS variable).
```
Comment (adamdehaven) [inline README.md:181]:
```suggestion
**Note:** notice how you have to use [interpolation](https://sass-lang.com/documentation/interpolation/) when using SCSS variable tokens in custom property values (such as defining your CSS variable).
```

File: README.md (hunk 1 above)
Comment (portikM) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (portikM) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -163,6 +163,17 @@ As an example, in Kong's [Kongponents](https://kongponents.konghq.com) Vue compo
...
 .my-component-class {
+  /**
+   * When setting a CSS variable value to a SCSS variable, you need to 
+   * interpolate the SCSS variable. Interpolation can be used 
+   * almost anywhere in a Sass stylesheet to embed the result of a 
+   * SassScript expression into a chunk of CSS. 
+   * Just wrap the expression in `#{}`
+  */
+  
+  --my-custom-scoped-css-variable: var(--kui-space-20, #{$kui-space-20});
```
Comment (adamdehaven) [inline README.md:174]:
```suggestion
  */
  --my-custom-scoped-css-variable: var(--kui-space-20, #{$kui-space-20});
```

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**